### PR TITLE
Fix #12701 Disable upgrade pop-up at slient install

### LINKED ISSUE #12701: Silent NSIS setup flag /S isn't silent, upgrade pop-up appears anyway
**OS:** Windows
**Version:** 0.3.0

**Steps to reproduce:**
1.  Pass `/S` to the setup installer executable to silently install/upgrade OpenRCT2
2. The [upgrade pop-up](https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/windows/install.nsi#L510) appears anyway, causing a non-zero (0) 1223 exit code.

**Tested with:**
* OpenRCT2-0.3.0-windows-installer-win32.exe
* OpenRCT2-0.3.0-windows-installer-x64.exe
```
---------------------------
OpenRCT2 0.3.0 32-bit for Windows Vista, 7, 8.1 and 10 Setup
---------------------------
Welcome to OpenRCT2 0.3.0 Setup.
This will allow you to upgrade from version 0.3.0.
---------------------------
OK   
---------------------------
```

### PR DESCRIPTION
**Pull request by Lastorder-DC**
If slient mode is activate, ignore previous version check and jump to FinishCallback.

Processor architecture warning(Installing 32bit version at 64bit windows, or vice versa) will still be shown since It's recommended to install 64bit version if windows version is 64bit, and installing 64bit version at  32bit windows will not work.

### CODE REVIEW

**Comment by IntelOrca** [comment]
Are you able to add an entry to the changelog?

**Comment by Lastorder-DC** [comment]
@IntelOrca Done.

**Comment by IntelOrca** [comment]
For bug fixes, we write the title of the bug report which in this case would be:
`Silent NSIS setup flag /S isn't silent, upgrade pop-up appears anyway`

This mentions NSIS which I think is enough.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Fix: [#12691] Ride graph tooltip incorrectly used count instead of number string.
 - Fix: [#12694] Crash when switching ride types with construction window open.
+- Fix: [#12701] Upgrade popup appears on silent mode.
```
Comment (AaronVanGeffen) [inline distribution/changelog.txt:5]:
```suggestion
- Fix: [#12701] Installer: upgrade popup appears on silent mode.
```

**Comment by Lastorder-DC** [comment]
@IntelOrca Fixed and force pushed.

File: distribution/changelog.txt [hunk 2]
```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Fix: [#12691] Ride graph tooltip incorrectly used count instead of number string.
 - Fix: [#12694] Crash when switching ride types with construction window open.
+- Fix: [#12701] Silent NSIS setup flag /S isn't silent, upgrade pop-up appears anyway
```
Comment (IntelOrca) [inline distribution/changelog.txt:5]:
```suggestion
- Fix: [#12701] Silent NSIS setup flag /S isn't silent, upgrade pop-up appears anyway.
```

**Pull request merged**